src/post/post.repository.knex.ts

```diff
@@ -430,99 +389,4 @@ export class KnexPostRepository {
             accountIds: newRepostAccountIds,
         };
     }
-
-    /**
-     * Insert replies associated with a post into the database
-     *
-     * @param replies Replies to insert
-     */
-    private async insertReplies(
-        replies: Post[],
-        transaction: Knex.Transaction,
-    ) {
-        const repliesToInsert = replies.map((reply) => ({
-            uuid: reply.uuid,
-            type: reply.type,
-            audience: reply.audience,
-            author_id: reply.author.id,
-            title: reply.title,
-            excerpt: reply.excerpt,
-            content: reply.content,
-            url: reply.url?.href,
-            image_url: reply.imageUrl?.href,
-            published_at: reply.publishedAt,
-            in_reply_to: reply.inReplyTo?.id,
-            thread_root: reply.threadRoot?.id,
-            ap_id: reply.apId.href,
-        }));
-
-        await transaction(TABLE_POSTS).insert(repliesToInsert);
-    }
-
-    /**
-     * Insert replies associated with a post into the database, ignoring
-     * duplicates
-     *
-     * @param replies Replies to insert
-     * @param transaction Database transaction to use
-     * @returns The number of replies inserted
-     */
-    private async insertRepliesIgnoringDuplicates(
-        replies: Post[],
-        transaction: Knex.Transaction,
-    ): Promise<number> {
-        const repliesToInsert = replies.map((reply) => ({
-            uuid: reply.uuid,
-            type: reply.type,
-            audience: reply.audience,
-            author_id: reply.author.id,
-            title: reply.title,
-            excerpt: reply.excerpt,
-            content: reply.content,
-            url: reply.url?.href,
-            image_url: reply.imageUrl?.href,
-            published_at: reply.publishedAt,
-            in_reply_to: reply.inReplyTo?.id,
-            thread_root: reply.threadRoot?.id,
-            ap_id: reply.apId.href,
-        }));
-
-        await transaction(TABLE_POSTS)
-            .insert(repliesToInsert)
-            .onConflict()
-            .ignore();
-
-        const [[{ count }]] = await transaction.raw(
-            'SELECT ROW_COUNT() as count',
-        );
-
-        return count;
-    }
-
-    /**
-     * Increment a count stored on a post by a given amount
-     *
-     * @param post Post to increment the count of
-     * @param countType Type of count to increment
-     * @param incrementBy Amount to increment the count by
-     * @param transaction Database transaction to use
-     */
-    private async incrementCount(
```

Comment:
This was only used in one place for each update type - I felt it is much easier to read by just inlining the call

Reply:
You can remove `PostCountType` as it should be unused now